src/System.Net.Primitives/tests/FunctionalTests/project.json

```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "System.Diagnostics.Contracts": "4.0.0",
     "System.Runtime": "4.0.20",
-    "System.Threading.Tasks": "4.0.0",
+    "System.Threading.Tasks": "4.0.11-rc2-23604",
+    "System.Threading.Thread": "4.0.0-rc2-23604",
+    "System.Reflection.Extensions": "4.0.1-rc2-23604",
```

Comment:
We shouldn't hardcode version specific build numbers like this, i.e. we shouldn't have the "-rc2-23604" part.